frontend/components/ClubPage/Tabs.js

```diff
@@ -32,26 +33,7 @@ export default props => (
         content: (
           <div>
             {props.club.members.length ? (
-              props.club.members.map((a, i) => (
-                <div className="media" key={i}>
-                  <div className="media-left">
-                    <figure className="has-background-light image is-48x48"></figure>
-                  </div>
-                  <div className="media-content">
-                    <p className="title is-4">{a.name || 'No Name'}</p>
-                    <p className="subtitle is-6">
-                      {a.email ? (
-                        <span>
-                          <a href={'mailto:' + a.email}>{a.email}</a> ({a.title}
-                          )
-                        </span>
-                      ) : (
-                        a.title
-                      )}
-                    </p>
-                  </div>
-                </div>
-              ))
+              props.club.members.map((a, i) => <MemberCard a={a} i={i} />)
```

Comment:
Two things here:
1. the `key` prop should be declared here rather than passed as a prop
2. the key should not be an index (because if the order of members changes for any reason the app would not diff verions of the dom correctly so a better convention to find some prop in the club member which is unique which we can use as the key

Reply:
Changed it to 'a.username'!